server/routes/confirmedOnReport/confirmedOnReport.ts

```diff
@@ -42,6 +48,16 @@ export default class ConfirmedOnReportRoutes {
       bannerHTML: `Your report number is: <br><strong>${adjudicationNumber}</strong>`,
       buttonClass: 'govuk-button--secondary',
       buttonHref: '/place-a-prisoner-on-report',
+      statement: adjudicationDetails.statement,
```

Comment:
I am missing something here - I expected this to have the same data as used by `confirmedOnReportChangeReport.ts`, but this has significantly more.

Prob subtle changes in the design that I don't know about